diff --git a/dist/src/template-tag.js b/dist/src/template-tag.js
index 5e46538aa54b1ebfc2df6358e7b62d7747cd695c..cac2759d7802a9d20c4d01cd19c5448b09384832 100644
--- a/dist/src/template-tag.js
+++ b/dist/src/template-tag.js
@@ -1,6 +1,19 @@
 import { createFilter } from '@rollup/pluginutils';
 import { Preprocessor } from 'content-tag';
-const gjsFilter = createFilter('**/*.{gjs,gts}?(\\?)*');
+const gjsFilter = createFilter(
+ // include: "*.gjs" with optional "?..."
+  '**/*.{gjs,gts}?([?]*)',
+
+  // exclude: any "gjs" anywhere BEFORE the final ".gjs"
+  [
+    '**/*gjs*/**', // any directory segment containing "gjs"
+    '**/*gts*/**', // any directory segment containing "gjs"
+    '**/*gjs*.gjs?([?]*)', // basename contains "gjs" before ".gjs" (also handles query)
+    '**/*gts*.gjs?([?]*)', // basename contains "gjs" before ".gjs" (also handles query)
+    '**/*gjs*.gts?([?]*)', // basename contains "gjs" before ".gjs" (also handles query)
+    '**/*gts*.gts?([?]*)', // basename contains "gjs" before ".gjs" (also handles query)
+  ]
+);
 export function templateTag() {
     let preprocessor = new Preprocessor();
     return {
