diff --git a/dist/compile/formats/gjs/index.js b/dist/compile/formats/gjs/index.js
index 2cd01b804084a314355adbfed311a95e1d5652a5..75a3cf034a259a5f59a5cccd738d2ddfac2424ac 100644
--- a/dist/compile/formats/gjs/index.js
+++ b/dist/compile/formats/gjs/index.js
@@ -1,4 +1,4 @@
-import * as compiler from 'ember-template-compiler';
+import * as compiler from 'ember-source/dist/ember-template-compiler.js';
 import { nameFor } from '../../utils.js';
 import { evalSnippet } from './eval.js';
 
